index.js

```diff
@@ -15,6 +15,11 @@ var PluginError = gutil.PluginError;
 var PLUGIN_NAME = 'gulp-istanbul';
 var COVERAGE_VARIABLE = '$$cov_' + new Date().getTime() + '$$';
 
+function getNormalizedPath(filepath) {
+  if (path.sep !== '\\') return filepath;
+  return filepath.replace(/\//g, '\\');
```

Comment:
That is true... I only added the conditional as conditional processing is probably faster than processing the regex across the file if not needed.